src/Database/Statement/SqliteStatement.php

```diff
@@ -28,18 +28,18 @@ class SqliteStatement extends StatementDecorator
     /**
      * @inheritDoc
      */
-    public function execute(?array $params = null): bool
-    {
-        if ($this->_statement instanceof BufferedStatement) {
-            $this->_statement = $this->_statement->getInnerStatement();
-        }
+    // public function execute(?array $params = null): bool
```

Comment:
This looks like it could be cleaned up.

Reply:
Yeah I'll deal with this in a separate PR.